pinot-core/src/main/java/org/apache/pinot/core/data/manager/realtime/PinotFSSegmentUploader.java

```diff
@@ -96,9 +96,8 @@ public URI uploadSegment(File segmentFile, LLCSegmentName segmentName) {
       LOGGER.info("Interrupted while waiting for segment upload of {} to {}.", segmentName, _segmentStoreUriStr);
       Thread.currentThread().interrupt();
     } catch (TimeoutException e) {
-      _serverMetrics.addMeteredTableValue(segmentName.getTableName(), ServerMeter.SEGMENT_UPLOAD_TIMEOUT, 1);
-      LOGGER.warn("Timed out waiting to upload segment: {} for table: {}",
-          segmentName.getSegmentName(), segmentName.getTableName());
+      _serverMetrics.addMeteredTableValue(rawTableName, ServerMeter.SEGMENT_UPLOAD_TIMEOUT, 1);
```

Comment:
@ankitsultana  Should we also emit metric for segment upload success / failure overall? This is not related to this PR.